naga/src/valid/interface.rs

```diff
@@ -67,6 +67,8 @@ pub enum VaryingError {
     MemberMissingBinding(u32),
     #[error("Multiple bindings at location {location} are present")]
     BindingCollision { location: u32 },
+    #[error("Multiple bindings use the same blend_src {blend_src}")]
```

Comment:
style: Let's use backticks here:

```suggestion
    #[error("Multiple bindings use the same `blend_src` {blend_src}")]
```

This feedback also applies to several other places referring to `blend_src` and other identifiers in code.

Reply:
thought: It's really annoying that, unlike WGSL frontend errors, I can't look at a validation error's text and have the labels that will _also_ be provided with that error close by.

todo: Let's file an issue to improve diagnostics here with spans of some sort. We should point the user to the first and second `blend_src`s that are claiming the same source slot.

Reply:
Filed https://github.com/gfx-rs/wgpu/issues/7299 and fixing lack of ticks